src/js/components/Ballot/PositionItem.jsx

```diff
@@ -20,19 +20,34 @@ export default class PositionItem extends Component {
 
     let image_placeholder = "";
     if (this.props.speaker_type === "O") {
-        image_placeholder = <i className="icon-org-lg icon-icon-org-placeholder-6-2 icon-org-resting-color"></i>;
+        image_placeholder = <i className="icon-org-lg icon-icon-org-placeholder-6-2 icon-org-resting-color position-item__avatar"></i>;
```

Comment:
I think BEM naming is a helpful way to organize styles – at least for right now. It forces us to focus on the semantics of what components do in a way that just using utility classes doesn't. Ideally, In the near term, I hope that it will help bring more clarity to and unify similar components. In the longer term, I hope to transition to a more React-specific hybridized approach with encapsulated styling, but using BEM naming seems like a good interim approach. Again, I'm still open to alternative suggestions.